packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/TableStories.tsx

```diff
@@ -40,18 +40,20 @@ function loadData(
   props: TableChartProps,
   { pageLength = 50, rows = 1042, cols = 8, alignPn = false, showCellBars = true },
 ): TableChartProps {
-  if (!props.queryData) return props;
-  const records = props.queryData?.data?.records || [];
-  const columns = props.queryData?.data?.columns || [];
+  if (!props.queriesData || !props.queriesData[0]) return props;
+  const records = props.queriesData?.[0].data?.records || [];
+  const columns = props.queriesData?.[0].data?.columns || [];
```

Comment:
I wonder if we need to check length here?
```
> const emptyArray = []
> emptyArray ? 'truthy' : 'falsy'
'truthy'
```

Reply:
amm... can you clarify please?

Reply:
Sorry, disregard, I read this too quickly and didn't notice the added `||` here.